ci/init-db.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 # Configure a set of databases in the database server for upgrade tests
 set -x
-for SUFFIX in '' _upgrade_072 _upgrade_081 _upgrade_094; do
+for SUFFIX in '' _upgrade_100 _upgrade_122 _upgrade_130; do
```

Comment:
Is it correct that this implies that you are required to upgrade to 1.* before upgrading to jupyterhub 2.* (assuming 2.0 is the next released version?)

Reply:
Yes, that's exactly right.

Reply:
Thanks!